graphkb/constants.py

```diff
@@ -56,8 +56,10 @@
 ONCOKB_SOURCE_NAME = 'oncokb'
 ONCOGENE = 'oncogenic'
 TUMOUR_SUPPRESSIVE = 'tumour suppressive'
-
 FUSION_NAMES = ['structural variant', 'fusion']
+PHARMACOGENOMIC_RELEVANCE_TERMS = ["decreased toxicity", "increased toxicity"]
```

Comment:
Hi @mattdoug604 , this constant was from gsc_report.

From the RELEVANCE_BASE_TERMS constant below, it seems like pharmacogenomic relevance terms should include metabolism and dosage.

`('pharmacogenomic', ['metabolism', 'toxicity', 'dosage']),`

Does that seem right, in general?  (There were probably just no metabolism or dosage GKB variants to count yet?)
Details on https://www.bcgsc.ca/jira/browse/SDEV-2733 might be relevant.